### DISCUSSION TITLE
Network unresponsiveness while using Nuclei

### DISCUSSION THREAD

**Discussion by offensiveevil** [projectdiscovery/nuclei#5595] [General]
Nuclei Version: 3.1.10

Current Behavior:
Whenever I run Nuclei, my network becomes unresponsive, and the internet connection ceases to work. This issue persists until I terminate the Nuclei scan. I'm using a Netis router, and the network interruption specifically occurs during the scan operation.

Expected Behavior:
I expected to run Nuclei for vulnerability scanning without it affecting my network's connectivity. The tool should perform its scans without rendering the internet connection unresponsive.

Steps To Reproduce:

Execute Nuclei with a standard template and target, e.g., nuclei -t <template> -u <target URL>.

Shortly after the scan starts, the network becomes unresponsive, and internet access is lost.

Terminate the Nuclei scan.

Network responsiveness and internet access are immediately restored.

Potential Areas for Investigation:

Rate Limiting and Configuration: Investigating if specific configurations or rate limiting options can be adjusted to prevent network overload.

Compatibility with Netis Routers: Whether there's a known compatibility issue with Netis routers that could be causing the network to become unresponsive.

I am looking for guidance on adjustments I can make to either Nuclei's configuration or my network setup to prevent this issue. Any advice or insights from the community would be greatly appreciated.

**Comment by offensiveevil** [comment]
Update: Seems like this does not only exist in that particular version of router. In some cases the host itself losses connection.

**Comment by KickedDroid** [comment]
> Update: Seems like this does not only exist in that particular version of router. In some cases the host itself losses connection.

This issue is occurring for me as well and I am not using that router or software. 

I'm running PopOS! Linux I'm wondering if it's possibly a Linux issue?

**Comment by offensiveevil** [comment]
Yeah. My colleague was facing the same issue and he uses linux too.

**Comment by Harsh-Navgale** [comment]
yes i am having the same issue

**Comment by thev1ctori0us** [comment]
I was facing the same issue these days. When im using Nuclei, my whole network turn down, and at the moment i stop Nuclei, the connection returns. Im using Kali Linux in WSL2. I also test in Ubuntu latest version in WSL2 and another Kali Linux in my laptop. Both doesnt work.

With my test, i declared that my network's ISP/firewall blocked my connection, assuming that Nuclei was trying to do a malicious request. Your problem could be different, or the cause may be different, but i belive that your problem are the same as me.

The only solution was use a VPN. U can try contact your network provider to analise this or add a firewall rule to not block your connection, or you can do this by yourself. I tried this, but my provider did not want to resolve the problem. In that case only try this by yourself if you know what you are doing!

So if you have this problem, i suggest u to try a VPN or a VPS.
Hope i helped you guys :)

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
